src/components/projectionError.module.css

```
@@ -0,0 +1,3 @@
+.ProjectionError:global(.ui.message) {
+  margin: 0 10%;
```

```suggestion
  margin: 0 20px;
```
Better work with absolute values.

---

done!